src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -60,8 +60,16 @@ public ListPanel(UI ui, IModel model, PanelControl parentPanelControl, int panel
      * @param issue
      * @return true if the issue has changed, false otherwise
      */
+
+    private static boolean qualifierNamesHaveUpdatedQualifier(List<String> filterQualifierNames){
+        // omitting self-updated issues since we only care about updated comments from other users.
+        return (filterQualifierNames.contains(Qualifier.UPDATED) ||
+                filterQualifierNames.contains(Qualifier.UPDATED_OTHERS));
+    }
```

Comment:
This should be moved to `Qualifier`, along with the code that is calling it. The whole thing can take in a `FilterExpression` and return a boolean value.